lib/infrastructure/external_api_broker.rb

```diff
@@ -0,0 +1,63 @@
+require "rest_client"
+
+class ExternalApiBroker
+  def get_catalog_name(catalog_id)
+    get_object_name(catalog_id, catalog_url)
+  end
+
+  def get_plan_name(plan_id)
+    get_object_name(plan_id, plan_url)
+  end
+
+  def order_service_plan(payload)
+    make_request(method: :post, url: order_service_plan_url, payload: payload)
+  end
+
+  private
+
+  def base_url
+    "https://localhost"
+  end
+
+  def catalog_url
+    URI.join(base_url, "apis/servicecatalog.k8s.io/v1beta1/clusterserviceclasses").to_s
+  end
+
+  def order_service_plan_url
+    URI.join(base_url, "apis/servicecatalog.k8s.io/v1beta1/namespaces/default/serviceinstances").to_s
+  end
+
+  def plan_url
+    URI.join(base_url, "apis/servicecatalog.k8s.io/v1beta1/clusterserviceplans").to_s
+  end
+
+  def make_request(method:, url:, headers: generic_headers, payload: nil)
+    request_options = {
+      :method     => method,
+      :url        => url,
+      :headers    => headers,
+      :verify_ssl => true
```

Comment:
We should add this to the default endpoint also

Reply:
We should probably go with something similar to `security_protocol` that we have on ManageIQ

Reply:
Yeah that's what I was thinking too, also a custom certificate_authority.  I'll put a PR in to add both of those to Endpoints

Reply:
Hey @eclarizio verify_ssl and certificate_authority were added to the Endpoints model in https://github.com/ManageIQ/topological_inventory-core/pull/42

Reply:
Thanks @agrare I'll change this too, then.